samples/wifi/shutdown/README.rst

```diff
@@ -28,6 +28,15 @@ The sample:
 #. Shuts down the Wi-Fi driver.
 #. Puts the nRF5340 SoC in the lowest possible power consumption mode.
 
+Shutdown mode
```

Comment:
Let's remove this title, as it is confusing with the related power state. If you want a separate section just use

Notes
******
 or any other generic name :)